src/lazy/text/buffer.rs

```diff
@@ -701,7 +722,7 @@ impl<'data> TextBufferView<'data> {
 
     /// Matches the three parts of an int--its base, its sign, and its digits--without actually
     /// constructing an Int from them.
-    fn match_int(self) -> IonParseResult<'data, MatchedInt> {
+    pub fn match_int(self) -> IonParseResult<'data, MatchedInt> {
```

Comment:
Just curious—why does this need to be `pub` now?

Reply:
I wanted each of the methods that matched a full Ion type to be reusable from outside the buffer (but not outside the crate without more consideration).